mtest/run_test.go

```diff
@@ -106,34 +105,43 @@ func runManagementEtcd(client *ssh.Client) error {
 }
 
 func stopCKE() error {
+	env := cmd.NewEnvironment(context.Background())
 	for _, host := range []string{host1, host2} {
-		c := sshClients[host]
-		sess, err := c.NewSession()
-		if err != nil {
-			return err
-		}
+		host2 := host
+		env.Go(func(ctx context.Context) error {
+			c := sshClients[host2]
+			sess, err := c.NewSession()
+			if err != nil {
+				return err
+			}
+			defer sess.Close()
 
-		sess.Run("sudo systemctl reset-failed cke.service; sudo systemctl stop cke.service")
-		sess.Close()
+			sess.Run("sudo systemctl reset-failed cke.service; sudo systemctl stop cke.service")
+
+			return nil // Ignore error if cke was not running
+		})
 	}
-	return nil
+	env.Stop()
+	return env.Wait()
 }
 
 func runCKE() error {
+	env := cmd.NewEnvironment(context.Background())
 	for _, host := range []string{host1, host2} {
-		c := sshClients[host]
-		sess, err := c.NewSession()
-		if err != nil {
-			return err
-		}
+		host2 := host
+		env.Go(func(ctx context.Context) error {
+			c := sshClients[host2]
+			sess, err := c.NewSession()
+			if err != nil {
+				return err
+			}
+			defer sess.Close()
 
-		err = sess.Run("sudo systemd-run --unit=cke.service --setenv=GOFAIL_HTTP=0.0.0.0:1234 /data/cke -config /etc/cke.yml -interval 10s -session-ttl 5s")
-		sess.Close()
-		if err != nil {
-			return err
-		}
+			return sess.Run("sudo systemd-run --unit=cke.service --setenv=GOFAIL_HTTP=0.0.0.0:1234 /data/cke -config /etc/cke.yml -interval 1ms -session-ttl 5s")
```

Comment:
`-interval` must be positive value

Reply:
Does that mean that CKE fails to start if `-interval` is 0?